core/threaded/scheduler_instance.h

```diff
@@ -165,6 +224,7 @@ bool init_sched_instance(
     }
     lf_mutex_unlock(&mutex);
 
+    // Shaokai: Why are these things in the critical section?
```

Comment:
This comment is odd because it's outside the critical section.

Reply:
Thanks for catching this silly typo of mine... At the time of writing, I meant to say `Why are these things NOT in the critical section?`